servers/rendering/renderer_rd/storage_rd/light_storage.h

```diff
@@ -48,6 +48,11 @@ class RenderDataRD;
 namespace RendererRD {
 
 class LightStorage : public RendererLightStorage {
+	GDCLASS(LightStorage, RendererLightStorage);
```

Comment:
Just for others, this is the only question mark in this solution.

The classes in the rendering engine are namespaced (`RendererRD::` and `GLES3::`) and thus do not have unique names. Now that we're exposing them through `ClassDB` we have a problem if we want to expose both the compatibility and render device storage classes because `ClassDB` doesn't support namespacing (yet).

It is likely that we'll only ever expose the rendering device storage classes. Otherwise we may need to rename the classes after all or talk to the script team about what it will take to add namespace support to `ClassDB`.